### PR TITLE
KAN-32 Send push notification

### PR DESCRIPTION
**Pull request by Isaacjbar**
### Feature / Bug Description  
When a mechanic is assigned to a card, we need to send a push notification to notify the user. The notification must include:  

- **Title**:  
  `Card [card_id] has been assigned to you.`  

- **Body**:  
  `You have a new card assigned.`  

- **Type**:  
  An empty string (`""`).  

Additionally, when a user's status is updated to **Inactive**, a push notification must be sent with the `notification_type` value set to **"CLOSE_SESSION"**.  

The text content for all notifications has been centralized in the `Strings.ts` file to maintain consistency.  

---

#### Changes to `updateCardMechanic` Method  
- Updated the notification **title** to dynamically include the `card ID`.  

#### Changes to `updateUser` Method in `users.service`  
- Added logic to send a **push notification** when the user's status is updated to **Inactive**.  
- The notification includes:  
   - `notification_type` with the value **"CLOSE_SESSION"**.  

#### Centralized Notification Constants  
- Centralized all text constants (title, description, and type) in the `Strings.ts` file.  

#### Type Conversion Fix  
- Ensured `card.id` is converted to a string before replacing placeholders to avoid type errors.  

---

### Notes  
- Added new constants to **`Strings.ts`**.  
- Fixed the type error with `replace` by converting `card.id` to a string.  
- Verified the changes through testing to ensure:  
   - Notifications are sent as expected.  

### Tickets
[TICKET] (https://one-sm.atlassian.net/browse/KAN-32)
[TICKET] (https://one-sm.atlassian.net/browse/KAN-33)

### Screenshots / Screencasts
n1
image
session

### CODE REVIEW

File: src/modules/users/users.service.ts [hunk 1]
```diff
@@ -363,10 +366,23 @@ export class UsersService {
       user.uploadCardDataWithDataNet = updateUserDTO.uploadCardDataWithDataNet;
       user.uploadCardEvidenceWithDataNet =
         updateUserDTO.uploadCardEvidenceWithDataNet;
+      user.status = updateUserDTO.status;  
       user.updatedAt = new Date();
 
       await this.userRepository.save(user);
 
+      if (updateUserDTO.status === 'I') {
```
Comment (imadiaz) [inline src/modules/users/users.service.ts:374]: can we use a constant fr this i?

Comment (Isaacjbar) [reply src/modules/users/users.service.ts]: OF course. Ready

Comment (Isaacjbar) [reply src/modules/users/users.service.ts]: image

**Review by XimenaPulidoGutierrez — APPROVED** [review]
LGTM

**Comment by ErickGS28** [comment]
lgtm

**Review by ErickGS28 — COMMENTED** [review]
lgtm

**Review by ErickGS28 — APPROVED** [review]
lgtm

**Pull request merged**